src/main/java/org/ladocuploader/app/data/enums/PersonalSituationType.java

```diff
@@ -0,0 +1,37 @@
+package org.ladocuploader.app.data.enums;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public enum PersonalSituationType {
+  HOMELESS_SWITCH("experiencing homelessness", "", "personal-situations.which.option1"),
+  DRUGS_ALCOHOL("struggling with drugs or alcohol", "", "personal-situations.which.option2"),
+  DOMESTIC_ABUSE("experiencing domestic abuse", "", "personal-situations.which.option3"),
+  DISABILITY("personal issue or disability", "", "personal-situations.which.option4"),
+  OTHER("other", "", "personal-situations.which.option5");
+
+  private final String value;
+  private final String abbreviation;
```

Comment:
You can take out the abbreviation altogether - as it's not needed here.  These fields are flexible and can be whatever you want or need.

Reply:
Let me know if you want to look at this together.